tgui/packages/tgui/interfaces/PreferencesMenu/preferences/species/hemophage.ts

```diff
@@ -0,0 +1,51 @@
+import { Species } from "./base";
+
+const Hemophage: Species = {
+  description: "A crewmember afflicted with Hemophagia. Whether it's supernatural or purely through science, is still unknown.",
+  features: {
+    good: [{
+      icon: "bed",
+      name: "Locker Brooding",
+      description: "Hemophages can delay The Thirst and heal by resting in a \
+        rectangular-shaped object. So THAT'S why they do that!",
+    }, {
+      icon: "skull",
+      name: "Minor Undead",
+      description: "Hemophage can delay their thirst and heal by resting in any \
+        locker. Useful, not entirely practical, though.",
```

Comment:
I feel like we should probably change this to something else to reflect their virus immunity and their lack of breathing.
Something like this:
```suggestion
      icon: "skull",
      name: "Viral Symbiosis",
      description: "Hemophage, due to their condition, cannot get infected by \
        other viruses and don't actually require an external source of oxygen \
        to stay alive.",
```